.devcontainer/post_start_command.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-cd api && pip install -r requirements.txt
\ No newline at end of file
+cd api && poetry install
```

Comment:
or just
```suggestion
poetry install -C api
```

Either version is good enough for me.